velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -772,11 +1110,21 @@ static bool compareOrderedPartitions(
       return false;
     }
   }
-  if (not compareMaterializedRows(left.second, right.second)) {
-    return false;
-  }
 
-  return true;
+  auto width = left.second.begin()->size();
+  auto [floatColumnIndices, nonFloatColumnIndices] =
+      getFloatColumnIndices(left.second, width);
+  MaterializedRowsForEpsilonComparison rowsForComparison{
+      floatColumnIndices, nonFloatColumnIndices};
+  if (!floatColumnIndices.empty() &&
+      rowsForComparison.prepareForEpsilonComparison(
```

Comment:
It looks like a call to prepareForEpsilonComparison is always followed by a call to resultsMatch. I wonder if it makes sense to change the signature of prepareForEpsilonComparison to return a boolean and remove resultsMatch.